website/docs/r/kubernetes_cluster.html.markdown

```diff
@@ -945,6 +945,10 @@ A `http_proxy_config` block supports the following:
 
 A `upgrade_settings` block supports the following:
 
+* `drain_timeout_in_minutes` - (Optional) The amount of time in minutes to wait on eviction of pods and graceful termination per node. This eviction wait time honors waiting on pod disruption budgets. If this time is exceeded, the upgrade fails.
```

Comment:
> This eviction wait time honors waiting on pod disruption budgets

I'm unsure if this is correct, in the context of deleting the cluster we actually ignore PDBs

Reply:
The text is more about upgrades. I would intuitively expect that it is ignored for cluster deletion.

Reply:
I haven't done much with PDBs so it wasn't implicit for me. I think to avoid any potential confusion it'd be good to 
specify that e.g. 
```suggestion
* `drain_timeout_in_minutes` - (Optional) The amount of time in minutes to wait on eviction of pods and graceful termination per node. This eviction wait time honors pod disruption budgets for upgrades. If this time is exceeded, the upgrade fails.
```